open_humans/templates/base.html

```diff
@@ -76,7 +76,11 @@
             <li class="{% active 'research' %}">
               <a href="/research/">Research</a>
             </li>
-
+            
+            <li class="{% active 'activities-list' %}">
+              <a href="{% url '#activities-list' %">Activites</a>
```

Comment:
I removed 'Activities' from the navbar when the list moved on to the 'home' page...

Is it a UX thing, users can't find their way back to the link on the home page after exploring other pages? (This makes sense to me. It hadn't occurred to me.)

Also, I haven't checked, but the code presumably barfs on line -- it's trying to reference a [named URL pattern](https://docs.djangoproject.com/en/1.10/topics/http/urls/#naming-url-patterns) that doesn't exist (i.e. '#activities-list' that isn't named in /open_humans/urls.py)

I think the correct link for this should be: `href="{% url 'home' %}#activities-list"`